add-ons/grafana-prometheus/README.md

```diff
@@ -0,0 +1,39 @@
+# Grafana Add-on
+
+This addon installs grafana for metrics visualization from prometheus
+
+## Prerequisites
+
+The prometheus addon is required
```

Comment:
It is a good practice to tell how much RAM and CPU needs to be allocated to Minishift for this to run smoothly.

Reply:
grafana requires very little memory and cpu. it's merely to display data from its configured backend
see [here](https://community.grafana.com/t/hardware-requirements-for-a-grafana-server/2853)